packages/assets/scss/03-organisms/_stacked-row-section.scss

```diff
@@ -1,102 +1,102 @@
-@use "00-base/configure" as *;
-
-.ma__stacked-row {
-  position: relative;
-
-
-  &__section {
-
-    & ~ & {
-
-      @media ($bp-large-max) {
-        border-top-width: 1px;
-        border-top-style: solid;
-      }
-
-      .ma__stacked-row__title .ma__comp-heading {
-        margin-top: 15px;
-      }
-    }
-
-    & ~ &--borderless {
-
-      @media ($bp-large-max) {
-        border-top-width: 0;
-      }
-    }
-
-    & .main-content .page-content > *,
-    & .main-content--full .page-content > * {
-
-      @include ma-component-spacing;
-    }
-
-    & .ma__social-links-bar > .main-content {
-      // override default .main-content bottom spacing
-      margin-bottom: 0px;
-    }
-
-  }
-
-  &__container {
-
-    @include ma-container;
-  }
-
-  // reduce the width of the title and border
-
-  &__section--restricted + &__section--restricted &__container {
-
-    @include ma-container('restricted');
-  }
-
-  &__section ~ &__section &__container:before {
-    content: "";
-    display: block;
-
-    @media ($bp-large-min) {
-      border-top-width: 1px;
-      border-top-style: solid;
-    }
-  }
-
-  &__section ~ &__section--borderless &__container:before {
-
-    @media ($bp-large-min) {
-      border-top-width: 0;
-    }
-  }
-
-  &__section--restricted &__title {
-    max-width: 820px;
-  }
-  
-  // margin adjustment for callout-message (feature message)
-
-  // & &__section:first-child .main-content .page-content > section:not(.ma__callout-message__container)
-
-  & &__section:first-child .main-content .page-content > .ma__featured-item-mosaic,
-  & &__section:first-child .main-content .page-content > .ma__contact-list {
-    margin-top: calc(0.45em + 22px);
-
-    @media ($bp-x-small-min) {
-      margin-bottom: calc(0.45em + 18px);
-    }
-  }
-}
-
-//theme
-
-.ma__stacked-row {
-
-  &__section {
-
-    & ~ & {
-      border-color: $c-bd-divider;
-
-      .ma__stacked-row__container:before {
-        border-color: $c-bd-divider;
-      }
-    }
-  }
-}
+@use "00-base/configure" as *;
```

Comment:
looks like this file is having the same diff issue again...

Reply:
I can't see the real diffs in the scss files

Reply:
In the case of page-banner.scss, here are the actual diffs: https://github.com/massgov/mayflower/pull/1537/commits/d5541fcfcb9105cd0779b0ee7b460eeabbeb409c#diff-9cecfa3b1644a90fe0e38e50295220015a34686722be82013af1f80a7ccd1a21

Reply:
This file has not changed, ignore the changes --> see this PR for scss diffs https://github.com/massgov/mayflower/pull/1589